### ISSUE TITLE
video call errors

### ISSUE THREAD

**Issue by judyklaus** [BelledonneCommunications/linphone-android#1309]
**Describe the bug**
Hello, I'm having trouble using the video call of Linphone Android. It is not stable and I occasionally I cannot turn on/off the camera during a call. I made a quick test and wrote down the steps below.

**To Reproduce**
1. App1 calls App2 and then turns the camera on.
2. App2 turns video off.
3. App1 turns video on and then turns it off.
4. App2 cannot turn the video on anymore.

**SDK logs**

App1 logs: https://www.linphone.org:444//tmp/604b4c36c94e0_fcae0d12e829357a3634.gz
App2 logs: https://www.linphone.org:444//tmp/604b4c367f317_520f7d4449a9c7d262a3.gz

**Comment by Viish** [comment]
Hi,

Seems like a network issue. 
But you are using a very old version of linphone: `User-Agent: LinphoneAndroid/4.4.0 (Venus V7) LinphoneSDK/4.3.5 (tags/4.3.5^0)`
Please update to latest release or even better to current beta, your problem is very likely to be resolved.

Cheers,

**Comment by judyklaus** [comment]
Hi,

I updated my versions. Now, the problem occurs faster, in the 3rd step. App1 cannot turn the video on in the 3rd step. Here are the logs.

https://www.linphone.org:444//tmp/604f06d9b62a7_ea63bb92f79e8aa888cd.gz
https://www.linphone.org:444//tmp/604f06e2eae51_a4c06a56b51768653e9a.gz

**Comment by judyklaus** [comment]
Hi @Viish,

I updated my versions. Now, the problem occurs faster, in the 3rd step. App1 cannot turn the video on in the 3rd step. Here are the logs.

https://www.linphone.org:444//tmp/604f06d9b62a7_ea63bb92f79e8aa888cd.gz
https://www.linphone.org:444//tmp/604f06e2eae51_a4c06a56b51768653e9a.gz

**Comment by Viish** [comment]
Let's keep this thread instead of creating a new one.
Please try disabling ICE and/or with the current public beta on the Play Store.

**Comment by judyklaus** [comment]
Hi @Viish I disabled ICE on both side, still the problem occurs.

App1 calls App2
App1 turns cam on then on
App2 cannot turn camera on

App1 logs: attached text file
[linphone_log.zip](https://github.com/BelledonneCommunications/linphone-android/files/6161661/linphone_log.zip)


App2 logs: https://www.linphone.org:444//tmp/6052ecd65bf99_7e21d994aae2f3da380f.gz

**Comment by Viish** [comment]
Please try to reproduce with public beta on the Play Store, I don't reproduce your issue.
Also the logs from App1 you attached in your latest post is empty.

**Comment by judyklaus** [comment]
Hi @Viish isn't the release version on Google Play the public beta version?

**Comment by Viish** [comment]
That depends if you joined the beta program on the Play Store or not.
You can see the version in the about page. Unless it is 4.4.0-beta-X, you aren't on the beta.

**Comment by judyklaus** [comment]
Okay no. I have 4.3.1 and I am not joined to the beta program. So, when will this version be publicly released?

**Comment by Viish** [comment]
Should happen soon, probably next week.

**Comment by judyklaus** [comment]
Hi @Viish I downloaded 4.4 beta release of Linphone Android. The problem still continues.

App1 is an android phone and App2 is android emulator. 
App1 calls App2. App1 turns cam on and then off. App2 cannot turn on camera. (No request is sent to App1)
(I muted the call intentionally)

App1: https://www.linphone.org:444//tmp/60546576e72b5_4991264d4d923c2aaabb.gz
App2: https://www.linphone.org:444//tmp/60546565917e4_efa2ab25b22c67923355.gz
